.github/workflows/ruby-gem-publication.yml

```diff
@@ -0,0 +1,12 @@
+name: Ruby Gem Publish
+
+on:
+  push:
+    tags: v*
```

Comment:
`rake bump` and `rake release` will bump, commit, tag, and push, then it can be canceled at the `rubygems.org` stage and this workflow will take over.

Reply:
added (contributing.md)